public/app/plugins/datasource/elasticsearch/query_builder.ts

```diff
@@ -355,6 +387,15 @@ export class ElasticQueryBuilder {
     return query;
   }
 
+  private toNumber(stringValue: unknown): unknown | number {
+    const parsedValue = parseInt(`${stringValue}`, 10);
```

Comment:
Similar questions as for alpha, beta, gamma settings in the backend - they should be floats 0-1 so I guess we shouldn't parse them to ints?

Reply:
updated 👍